### PR TITLE
Make aita blog post

### PR DESCRIPTION
**Pull request by andronovhopf**

### CODE REVIEW

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+Since this dataset will be updated regularly, we’re using git and DVC to
+package, version, and release it. The data itself is stored in an S3 bucket, and
+you can use DVC to import the data to your workspace. If you haven't already
+you'll need to [install DVC](https://dvc.org/doc/install); one of the simplest
+ways is `pip install dvc`.
+
+Say you have a directory on your local machine where you plan to build some
+analysis scripts. Simply run
+
+```
```
Comment (shcheklein) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: all CLI blocks should start with "```dvc" to enable syntax highlighting

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+## A quick look at the data
+
+Let’s take a flyover look at the dataset so far. The code to make the following
+visuals and results is
+[available on GitHub](https://github.com/andronovhopf/aita_viz_and_classify)
+First, here’s a frequency plot for how common different verdicts are on the
+subreddit. In addition to “Asshole” and “Not the Asshole”, there are two
+additional rulings: “Everybody Sucks” and “No Assholes Here”.
+
+![](/uploads/images/2020-02-17/freq_plot.svg 'freqplot =1000')
```
Comment (shcheklein) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: what happens if `'freqplot =1000'` is not specified?

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 3]
```diff
@@ -0,0 +1,324 @@
...
+title:
+  AITA for making this? A public dataset of Reddit posts about moral dilemmas
+date: 2020-02-17
+description: |
+  Releasing an open natural language dataset based on r/AmItheAsshole.
+descriptionLong: |
+  Delve into an open natural language dataset of posts about tricky situations from
+  r/AmItheAsshole. Use this dataset for whatever you want- we'll show you how to get it and
+  start playing.
+picture: ../../static/uploads/images/2020-02-17/header.png
```
Comment (shcheklein) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: image looks blurry on the blog post page

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 4]
```diff
@@ -0,0 +1,324 @@
...
+$ dvc get --rev lightweight \
+      https://github.com/iterative/aita_dataset \
+      aita_clean.csv
+```
+
+## A quick look at the data
+
+Let’s take a flyover look at the dataset so far. The code to make the following
+visuals and results is
+[available on GitHub](https://github.com/andronovhopf/aita_viz_and_classify)
```
Comment (shcheklein) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: . is missing at the sentence end?

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 5]
```diff
@@ -0,0 +1,324 @@
...
+r/AmItheAsshole is a semi-structured online forum that’s the internet’s closest
+approximation of a judicial system. In this corner of the web, citizens post
+situations from their lives and Redditors vote to decide if the writer has acted
+as The Asshole or not. For example:
+
+![](/uploads/images/2020-02-17/aita_sample.png)
+
+Without bringing any code into the picture, it’s intuitive to think of each new
+post as a classification task for the subreddit. Formally, we could think of the
+subreddit as executing a function f such that
```
Comment (shcheklein) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: f -> `f`

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 6]
```diff
@@ -0,0 +1,324 @@
...
+
+![](/uploads/images/2020-02-17/aita_formula.png)
```
Comment (shcheklein) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: make is smaller with `=400`?

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: How do you see the blog review live?

Comment (shcheklein) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: Screen Shot 2020-02-17 at 4 06 36 PM

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 4 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: yep, fixing that!

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 6 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: I'll try that

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 5 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: I think it might be more proper to italicize f, since that's standard for mathematical notation. In any case it shouldn't be the same formatting as the rest of the sentence

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 3 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: @shcheklein hm.... I can't make it much higher resolution since the problem is actually that I can't get larger pictures of the emojis (they don't seem to come in vector graphics, only .png). Any ideas how to get it sharper?

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 5 above)
Comment (shcheklein) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: yep, italic sounds good!

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 2 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: hmm actually nothing. earlier i had a pretty small .png file and this needed to be specified or the image showed up tiny, but since I replaced with a .svg this size flag no longer seems necessary.

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 6 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: I like it at 500 px gonna do that

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 3 above)
Comment (shcheklein) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: you can copy emoji as an unicode symbol from this list - https://unicode.org/emoji/charts/full-emoji-list.html ... just copy paste it as a regular text into text box and make any size you'd like

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 7]
```diff
@@ -0,0 +1,324 @@
+---
+title:
+  AITA for making this? A public dataset of Reddit posts about moral dilemmas
+date: 2020-02-17
+description: |
+  Releasing an open natural language dataset based on r/AmItheAsshole.
+descriptionLong: |
+  Delve into an open natural language dataset of posts about tricky situations from
+  r/AmItheAsshole. Use this dataset for whatever you want- we'll show you how to get it and
+  start playing.
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: tricky situations -> moral dilemmas? For keyword consistency

we'll show -> I'll show? You use I throughout the text below.

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 8]
```diff
@@ -0,0 +1,324 @@
...
+  - Project
+  - Data
+  - DVC
+  - Reddit
+---
+
+In data science, we frequently deal with classification problems like, _is this
+[Yelp reviewer unhappy](https://www.ics.uci.edu/~vpsaini/) with their brunch? Is
+this [email begging me](https://archive.ics.uci.edu/ml/datasets/spambase) to
+claim my long-lost inheritance spam? Does this
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: Minor: I would link `[this email]` in this sentence.

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 9]
```diff
@@ -0,0 +1,324 @@
...
+[movie critic](http://ai.stanford.edu/~amaas/data/sentiment/) have a positive
+opinion of [Cats](https://bit.ly/2UXhaSW)?_
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: Minor: not sure you need to link _Cats_, maybe just italic?

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 10]
```diff
@@ -0,0 +1,324 @@
...
+
+Perhaps we should also consider the fundamental introspective matter of, _am I
+maybe being a bit of an asshole?_
+
+I want to share a dataset of collected moral dilemmas shared on Reddit, as well
+as the judgments handed down by a jury of Redditors. The wellspring of this data
+is the [r/AmItheAsshole](https://www.reddit.com/r/AmItheAsshole/) subreddit, one
+of the natural wonders of the digital world. In this article, I'll show you
+what's in the dataset, how to get it, and some things you can do to move the
+frontiers of Asshole research forwards.
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: forward

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 11]
```diff
@@ -0,0 +1,324 @@
...
+
+## What makes an Asshole?
+
+r/AmItheAsshole is a semi-structured online forum that’s the internet’s closest
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: Minor: are we using special quote characters in all blog posts? Which one is correct or easier to type?
```suggestion
r/AmItheAsshole is a semi-structured online forum that's the internet's closest
```

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 5 above)
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: or symbol ƒ

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 3 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: hmmmm even in a text editor the max resolution seems to be maxed out by the time I get ~300pt font 
Screen Shot 2020-02-17 at 4 27 42 PM

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 12]
```diff
@@ -0,0 +1,324 @@
...
+
+There are also many times I find myself disagreeing with the subreddit’s
+verdict. All this is to say, I don’t think it’s obvious how well a given human
+would do on the task of predicting whether Redditors find someone an Asshole.
+Nor is it clear how well we could ever hope for a machine to do approximating
+their judgment.
+
+It seems fun to try, though. It helps that the data is plentiful: because the
+subreddit is popular and well-moderated, there’s an especially strong volume of
+high-quality (re: on-topic and appropriately formatted) content being posted
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]:
```suggestion
high-quality content (re: on-topic and appropriately formatted) being posted
```

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 9 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: @jorgeorpinel i was worried about the case where a reader doesn't know what it is (is this movie known internationally?) if you think it's safe I'll delete the link

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 11 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: it's easier for me to type the one that's already in there- let me know if we need to commit to one

Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: interestingly, `’` renders when I type in Visual Studio Code, but `'` renders when I type here.

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 9 above)
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: Well you're talking about a movie critic, so it's given from that context 🙂 
Cats is known abroad, yes.

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 5 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: I tried the symbol but in our font (Brandon Grotesque) an italic f looks a lot closer to the math notation than the symbol. So sticking to italic

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 11 above)
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: I don't know. What do you think @shcheklein ? I guess since it's a blog each author will have their own style anyway.

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 3 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: is it possible to make the image a bit smaller in the header?

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 10 above)
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: Resolved without answer or change?

Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: I see both are used out there.

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 7 above)
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: Resolved without answer or change?

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 11 above)
Comment (shcheklein) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: @jorgeorpinel @andronovhopf don't have any strong opinion on this. Don't think it is that important to be honest.

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 9 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: alright goodbye link!

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 7 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: ah forgot to comment- changed tricky situations => moral dilemmas. Description now reads 
```Use this dataset for whatever you want- here's how to get it and
  start playing.```

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 3 above)
Comment (shcheklein) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: can we find svg/high res pngs in google for those? quite strange problem to have to be honest - no emojis in high res :)

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 7 above)
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: Ah sorry it had not changed when I checked. I guess you resolved before pushing. Np thanks

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 11 above)
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: Agree

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 12 above)
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: Also marked resolved but I don't see the change yet, maybe unpushed. Anyway, it reads better with parenthesis after content I think, no?

Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: i.e. it explains "high-quality content" too and this way it doesn't interrupt that phrase.

Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: Also not sure what "re:" means here but that's minor.

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 3 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: yeah apparently apple doesn't release official .svgs? this is never the problem i expected to have. 
So I tried shrinking the emojis I have down so they don't appear as blurry (also, the background behind the poop emoji is intentionally blurred). I just pushed this commit
Screen Shot 2020-02-17 at 5 07 37 PM

@shcheklein if you still think this is too blurry I will buy [these cheap .svgs](https://emojiisland.com/products/baby-angel-iphone-emoji-image for). I'm sorta attached to the baby face we have already but I'd be willing to switch

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 12 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: was unpushed, should be there now!

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 3 above)
Comment (shcheklein) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: buying if possible is a perfect option!

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 13]
```diff
@@ -0,0 +1,324 @@
...
+information entirely, we’re doing quite well! Although I can’t be certain, I’m
+curious how much the discrepancy comes down to dataset size: the previous effort
+with BERT appears to be trained on ~30K posts.
+
+Seeing that logistic regression on word counts doesn’t produce total garbage, I
+looked at which words were predictive of class using the
+[chi-squared test](https://scikit-learn.org/stable/modules/generated/sklearn.feature_selection.chi2.html).
+The top five informative words were mom, wife, mother, edit, and dad (looks like
+Assholes go back to edit their posts). Since familial relationships featured
+prominently, I estimated the log odds of being voted Asshole if your post
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: Consider explaining or linking "log odds" and other technical terms

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 12 above)
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: Cool, thanks

File: content/blog/2019-08-01-july-19-dvc-heartbeat.md [hunk 14]
```diff
@@ -11,7 +11,7 @@ descriptionLong: |
   Some of those are related to our brainchild DVC and its journey. The others
   are a collection of exciting stories and ideas centered around ML best
   practices and workflow.
-picture: ../../static/uploads/images/2019-08-01/post-image.png
+picture: ../../static/uploads/images/2019-08-01/AITA_post_header.png
```
Comment (shcheklein) [inline content/blog/2019-08-01-july-19-dvc-heartbeat.md]: ?

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 13 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: good idea, have added some more exposition and a link to a more detailed example from a tutorial i like

File: content/blog/2019-08-01-july-19-dvc-heartbeat.md (hunk 14 above)
Comment (andronovhopf) [reply content/blog/2019-08-01-july-19-dvc-heartbeat.md]: edited the wrong file on accident. that will be corrected.

File: content/blog/2020-02-17-a-public-reddit-dataset.md (hunk 3 above)
Comment (andronovhopf) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: remade with .svgs in the latest push

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 15]
```diff
@@ -0,0 +1,328 @@
...
+
+![](/uploads/images/2020-02-17/aita_sample.png)
+
+Without bringing any code into the picture, it’s intuitive to think of each new
+post as a classification task for the subreddit. Formally, we could think of the
+subreddit as executing a function _f_ such that
+
+![](/uploads/images/2020-02-17/aita_formula.png '=500')
+
+Of course, finding f won’t be trivial. To be frank, I’m not positive how well we
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: `finding _f_`

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 16]
```diff
@@ -0,0 +1,328 @@
...
+could hope to forecast the rulings of the subreddit. A lot of posts are not easy
+for me to decide- like,
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: Not sure about space usage around dash, and maybe it should be `–` em dash? https://www.thepunctuationguide.com/em-dash.html

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 17]
```diff
@@ -0,0 +1,328 @@
...
+
+As I said earlier, the plan is to grow the dataset over time. I’ve just run a
+new scrape for posts from January 1-31, 2020 and am adding them to the public
+dataset now. To check for a new release, you can re-run the `dvc get` command
+you used to grab the dataset.
+
+If you’re serious about taking on a project such as, say, building a classifier
+that beats our state of the art, word-count-based, logistic regression model,
+I’d like to recommend a better way to integrate the dataset into your workflow:
+`dvc import`. `dvc import` is like `dvc get`, but it preserves a link to the
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: To avoid some repetition here, I would rephrase like

`I’d recommend `dvc import` as a better way to integrate the dataset into your workflow. Importing also downloads the data, but it also preserves...`

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 18]
```diff
@@ -0,0 +1,328 @@
...
+you’ll want to re-train a model . To get the dataset the first time, you’ll run:
+
+```dvc
+$ git init
+$ dvc init
+$ dvc import https://github.com/iterative/aita_dataset \
+      aita_clean.csv
+```
+
+Then, because the dataset in your workspace is linked to our dataset repository,
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: `Then` -> `Later`?
Just not sure it's clear you're talking about future advantages of importing here, and not something the user can do right now.
p.s. don't worry, Ivan and I had review after review on the import and update command reference examples due to confusions like this one 😆 pretty tricky topic!

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 19]
```diff
@@ -0,0 +1,328 @@
...
+you can update it by simply running:
+
+```dvc
+$ dvc update aita_clean.csv
+```
+
+An additional benefit of codifying the link between your copy of the dataset and
+ours is that you can track the form of the dataset you used at different points
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: I'd use

`...and ours using `dvc import`, is that...`

Just to be extra clear what we're talking about.

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 20]
```diff
@@ -0,0 +1,328 @@
...
+```dvc
+$ dvc get https://github.com/iterative/aita_dataset \
+      aita_clean.csv
+```
+
+This will download a .csv dataset into your local directory, corresponding to
+the cleaned version. If you wanted the raw dataset, you would substitute
+`aita_raw.csv` for `aita_clean.csv`.
+
+Because the dataset is >100 MB, I’ve created a git branch (called “lightweight”)
```
[the remark this answers is not in the document]
Comment (jorgeorpinel) [reply content/blog/2020-02-17-a-public-reddit-dataset.md]: p.s. if the repo used `dvc run` stage files, the code sample further down in this doc (**How to get more data** section),

```console
$ git checkout 0de4fc3
$ dvc checkout
$ python train_classifier.py
```

could be revised to use `dvc repro` as well.

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 21]
```diff
@@ -0,0 +1,328 @@
...
+
+```dvc
+$ dvc update aita_clean.csv
+```
+
+An additional benefit of codifying the link between your copy of the dataset and
+ours is that you can track the form of the dataset you used at different points
+in your project development. You can jump back and forth through the project
+history then, not only to previous versions of code but also to versions of
+(specifically, links to) data. For example, you could roll back the state of the
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: Remove `then` and `(specifically, links to)`?
I found that sentence slightly long and unclear. Had to reread it a few times.

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 22]
```diff
@@ -0,0 +1,328 @@
...
+55031b0 first commit
+
+$ git checkout 0de4fc3
+$ dvc checkout
+$ python train_classifier.py
+```
+
+Oh, and one more note: you can always use `dvc get` and `dvc import` to grab an
+older version of the dataset using the tags associated with each release. The
+current release is v.20.1 and the original release is v.20.0- the numeric codes
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: `original release was`?

File: content/blog/2020-02-17-a-public-reddit-dataset.md [hunk 23]
```diff
@@ -0,0 +1,328 @@
...
+
+I expect that leaning into the particulars of the dataset- thinking about how
+the format influences the content, and how a subreddit might select for
+participants that don’t fully represent the population at large- will lead to
+more interesting questions than, say, aiming to forecast something about
+morality in general. To put it another way, the data’s not unbiased- so maybe
+try to learn something about those biases.
+
+If you make something with this dataset, please share- perhaps we can form an
+international Asshole research collective, or at least keep each other appraised
```
Comment (jorgeorpinel) [inline content/blog/2020-02-17-a-public-reddit-dataset.md]: 🤣

**Comment by jorgeorpinel** [comment]
Oh and a last comment is that maybe `https://blog.dvc.org/aita-public-reddit-dataset` would be a slightly better URL (using `aita` keyword for SEO?)
